packages/core/http/core-http-server-internal/src/http_server.ts

```diff
@@ -93,7 +95,8 @@ function startEluMeasurement<T>(
     if (
       eluMonitorOptions.logging.enabled &&
       active >= eluMonitorOptions.logging.threshold.ela &&
-      utilization >= eluMonitorOptions.logging.threshold.elu
+      utilization >= eluMonitorOptions.logging.threshold.elu &&
+      !JS_AND_MAP_REGEX.test(path)
```

Comment:
a little nitpicky, but can we simplify with endsWith()?

Reply:
Changed the code to use `endsWith`.
Even if faster it needs to check twice in the worst case now vs the regexp version, but I guess the path string won't grow too much nor the overhead would be too high in this case